src/JsonModel/JsonModel.js

```diff
@@ -515,7 +519,7 @@ class JsonModelImpl {
 			noCursor: true,
 		})
 		const row = await this.db.get(q, vals)
-		return this.toObj(row, options)
+		return this.toObj(row)
```

Comment:
This should be reverted, no? Options can be used to select columns

Reply:
right, it was even my change 🙈  I had to do some rebasing and somehow this git in, just fixed